azurerm/provider.go

```diff
@@ -190,6 +190,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_servicebus_namespace":                resourceArmServiceBusNamespace(),
 			"azurerm_servicebus_queue":                    resourceArmServiceBusQueue(),
 			"azurerm_servicebus_subscription":             resourceArmServiceBusSubscription(),
+			"azurerm_servicebus_rule":                     resourceArmServiceBusRule(),
```

Comment:
Hi Tom

I did consider this but in both the Service Bus documentation and in the go SDK they are known simply as 'rules' and i wanted to keep consistency with this terraform module.

https://docs.microsoft.com/en-us/rest/api/servicebus/rules/createorupdate

https://github.com/Azure/azure-sdk-for-go/blob/56332fec5b308fbb6615fa1af6117394cdba186d/services/servicebus/mgmt/2017-04-01/servicebus/rules.go

Reply:
so the Documentation and Go SDK are actually both generated from the same Swagger file - so the Azure Portal tends to be a more reliable source of truth. Given we've already got support for ServiceBus Topic Authorization Rules, I think this would be best as `Subscription Rule` (given it's scoped to that level) in-case a Namespace-wide rule is added in the future - WDYT?

Reply:
Ok, sounds good to me. Will get this sorted and updated soon